docs/src/documentation/03-components/08-visuals/seat/03-accessibility.mdx

```diff
@@ -51,10 +86,49 @@ Alternatively, the paragraph element with the id `l1` is visually hidden, so tha
 
 It is also recommended to have those strings translated and change dynamically based on the state of the user journey (eg: if the seat is selected and the user is about to deselect it, the screen reader should announce it).
 
-## SeatLegend
+#### Selected Seat with Price
 
-The SeatLegend component is not interactive. However, it accepts the `aria-label` prop, that is passed to the rendered SVG element.
+```jsx
+<Seat
+  title="Extra legroom seat 1C"
+  description="Aisle seat with extra legroom"
+  type="legroom"
+  selected
+  price="€15"
+  onClick={() => handleSeatSelection("1C")}
+/>
+```
 
-It allows for screen readers to provide a meaningful description of the seat type, which can be useful for users of assistive technologies.
+Screen reader announces: "Extra legroom seat 1C, Aisle seat with extra legroom, button, pressed".
 
-The `label` prop is also announced by screen readers.
+#### Unavailable Seat
+
+```jsx
+<Seat title="Seat 24B already occupied" type="unavailable" />
+```
+
+Screen reader announces: "Seat 24B already occupied, button, unavailable".
```

Comment:
Should we mention that this seat is not focusable because it does not have onClick? 🤔 Also, it is not announced like this to me. It never says "unavailable", but it does say "dimmed"
<img width="1292" alt="image" src="https://github.com/user-attachments/assets/f758b6ed-b760-4118-98cc-4926ac2ab84b" />